components/feature/syncedtabs/src/test/java/mozilla/components/feature/syncedtabs/storage/SyncedTabsStorageTest.kt

```diff
@@ -75,7 +81,7 @@ class SyncedTabsStorageTest {
         // This action will change the state due to lastUsed timestamp, but will run the flow.
         store.dispatch(TabListAction.RemoveAllPrivateTabsAction).joinBlocking()
 
-        verify(tabsStorage).store(
+        verify(tabsStorage, atLeastOnce()).store(
```

Comment:
Avoid using an unbound verification because future changes to the flow could lead to more unexpected calls and we could catch them here, if so.

We know that we should expect 2 calls - initial and updated states. So let's verify that.

```suggestion
        verify(tabsStorage, times(2)).store(
```

Reply:
Good point! I updated this here and below